### ISSUE TITLE
🆕 Software Suggestion | Runkod

### ISSUE THREAD

**Issue by wuniversales** [privacytoolsIO/privacytools.io#1644]
Name: Runkod
Category: Hosting descentralized
URL: https://runkod.com

Best hosting for static sites...

**Comment by Mikaela** [comment]
What makes them the best and deserve listing?

**Comment by wuniversales** [comment]
It is a decentralized hosting, based on blockstack, there are few decentralized, unlimited, encrypted and free hosting for static websites.
The only thing I can think of is ipfs, zeronet and i2p, but they require additional software, or failing that a kind of "Proxy" to load the web, and believe me, I have had ipfs websites, and most of the time, The websites load very slowly.
Instead, with this method, the websites work incredibly fast, and we ensure that it is decentralized through blockchain.

**Comment by lrq3000** [comment]
I did not test and did not know Runkod, but if it works as promised, that's indeed a great suggestion! I have previously looked at decentralized opensource hosting that could be accessible on the clearnet without any plugin, and the only thing I found was using Beaker Browser (based on DAT) plus hashbase.io to generate a clearnet address, but it seems way less elegant than Runkod which allows to setup any domain if I understood correctly, just like any other website.

The only thing missing is the support for dynamic server-side code websites, but from what I have read this is still being researched.

**Comment by Mikaela** [comment]
@wuniversales @lrq3000 Do you have any connection with Runkod?

**Comment by lrq3000** [comment]
Nope, no connection at all, you can look at my repo, I'm working in a
totally different domain (neuroscience). I just happen to share indieweb
vision that self or decentralized hosting is a must have for a free
internet :-)

**Comment by lrq3000** [comment]
Ah! There is another contender: OnionShare. Since v2.2, it supports a [Website Mode](https://github.com/micahflee/onionshare/wiki/Website-Mode) which allows to anonymously and securely publish a static website.

**Comment by wuniversales** [comment]
> @wuniversales @lrq3000 Do you have any connection with Runkod?

No, I have nothing to do with the project either.
I just saw it interesting, and currently I use it for hosting my websites.
https://buscatope.runkodapps.com/

By the way, as @lrq3000 says they are working to somehow create dynamic websites, I leave here some information.

https://steemit.com/blockstack/@runkod/runkod-road-ahead

**Comment by lrq3000** [comment]
@wuniversales Oh, thanks for the link, indeed if they can make dynamic websites work this would be a world-first in the domain as far as I know, certainly worthy of addition if they can retain their privacy features with this type of hosting!
